test/messages_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule MessageTest do
+  use ExUnit.Case
+
+  doctest Message
+
+  test "return welcome message" do
+    assert %Message{}.welcome === """
+                                  Tic Tac Toe
+                                  -----------
+                                  Instructions:
+                                  Select the spot you want to mark by entering the corresponding number.
+                                  The first to get three marks in a row will win the game.
+
+                                  You are player "X", please start the game.
+                                  """
+  end
+
+  test "return invalid input message" do
+    assert %Message{}.invalid === """
+                                  Please enter a number between 0-8.
+                                  """
+  end
+
+  test "return spot taken message" do
+    assert %Message{}.spot_taken === """
+                                  Spot is taken.
+                                  Please try again at a different spot.
+                                  """
+  end
+
+  test "return Computer win message" do
+    assert %Message{}.win === """
+                                  Computer Wins!!!
+                                  """
+  end
+
+  test "return game tie message" do
+    assert %Message{}.tie === """
+                                  It's a Tie!!!
+                                  """
+  end
+end
```

Comment:
I would argue that this isn't a necessary test file. To me, `Message` is more of a config file than a file that contains actual game logic. In theory, we should be able to change config files without breaking any tests.

Reply:
Agreed - there is a circumstance where you could reintroduce tests on files like this (Maybe for localization) - but as-is, it's probably OK to leave the tests out here.